.github/workflows/relay.yml

```diff
@@ -109,3 +109,58 @@ jobs:
         run: |
           rm -rf /tmp/.buildx-relay-cache
           mv /tmp/.buildx-relay-cache-new /tmp/.buildx-relay-cache
+
+  relay-build-and-publish-initcontainer:
+    if: github.event_name == 'workflow_dispatch'
```

Comment:
What do you think about enforcing similar rules as for `relay-build-test-publish`, so build without publishing on PR, and publish on merge or workflow_dispatch to `main`?
But we can handle it in a separate PR, as changes in this PR are an improvement anyways!

Reply:
Generally agree, but I think we can ditch the condition for publishing only for `main` when triggering the workflow manually (see my comment above).